privacysandbox/build-system
# Release 0.8.0

Release 0.8.0


### Features

* Add bash and jq to test-tools image
* Add script to normalize bazel- symlinks
* Ensure run-tests includes all images
* Skip symlinks that resolve in normalize-bazel-symlink

Change-Id: If04e1b0f5335c72d3712ab55eaf156d188915a47
